tests/integration_test/integration_test.sh

```diff
@@ -44,408 +44,573 @@
+    if [ "$1" == "info" ];
+    then
+        printf '\e[1;90m%-6s\e[m\n' "$2" # print gray
+    elif [ "$1" == "error" ];
+    then
+        printf '\e[1;91m%-6s\e[m\n' "$2"  # print red
+    elif [ "$1" == "success" ];
+    then
+        printf '\e[1;92m%-6s\e[m\n' "$2" # print green
+    elif [ "$1" == "warn" ];
+    then
+        printf '\e[1;93m%-6s\e[m\n' "$2" # print yellow
+    else
+        printf "$1"
+    fi
+
+}
+
+print_style "info" "Starting integration test"
+print_style "info" $(date +"%Y-%m-%d %H:%M:%S")
 
 set -e
 
-env=$1
-lira_mode=$2
-lira_version=$3
-pipeline_tools_mode=$4
-pipeline_tools_version=$5
-tenx_mode=$6
-tenx_version=$7
-ss2_mode=$8
-ss2_version=$9
-tenx_sub_id=${10}
-ss2_sub_id=${11}
-vault_token_path=${12}
-submit_wdl_dir=${13}
-use_caas=${14}
-use_hmac=${15}
-caas_collection_name=${16:-"lira-${env}-workflows"}
-
-work_dir=$(pwd)
-script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-
-printf "\n\nenv: $env"
-printf "\nlira_mode: $lira_mode"
-printf "\nlira_version: $lira_version"
-printf "\npipeline_tools_mode: $pipeline_tools_mode"
-printf "\npipeline_tools_version: $pipeline_tools_version"
-printf "\ntenx_mode: $tenx_mode"
-printf "\ntenx_version: $tenx_version"
-printf "\nss2_mode: $ss2_mode"
-printf "\nss2_version: $ss2_version"
-printf "\ntenx_sub_id: $tenx_sub_id"
-printf "\nss2_sub_id: $ss2_sub_id"
-printf "\nvault_token_path: $vault_token_path"
-printf "\nsubmit_wdl_directory: $submit_wdl_dir"
-printf "\nuse_caas: $use_caas"
-printf "\nuse_hmac: $use_hmac"
-printf "\ncaas_collection_name: $caas_collection_name"
-
-printf "\n\nWorking directory: $work_dir"
-printf "\nScript directory: $script_dir"
+LIRA_ENVIRONMENT=${1}
+LIRA_MODE=${2}
+LIRA_VERSION=${3}
+PIPELINE_TOOLS_MODE=${4}
+PIPELINE_TOOLS_VERSION=${5}
+TENX_MODE=${6}
+TENX_VERSION=${7}
+SS2_MODE=${8}
+SS2_VERSION=${9}
+TENX_SUBSCRIPTION_ID=${10}
+SS2_SUBSCRIPTION_ID=${11:-"placeholder_ss2_subscription_id"}
+VAULT_TOKEN_PATH=${12}
+SUBMIT_WDL_DIR=${13}
+USE_CAAS=${14}
+USE_HMAC=${15}
+COLLECTION_NAME=${16:-"lira-${LIRA_ENVIRONMENT}"}
+
+WORK_DIR=$(pwd)
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+GCLOUD_PROJECT=${GCLOUD_PROJECT:-"broad-dsde-mint-${LIRA_ENVIRONMENT}"} # other envs - broad-dsde-mint-test, broad-dsde-mint-staging, hca-dcp-pipelines-prod
+
+CAAS_ENVIRONMENT="caas-prod"
+LIRA_CONFIG_FILE="lira-config.json"
+# LIRA_DOCKER_TAG=${LIRA_DOCKER_TAG:-"ra_update_to_caas_prod"}
+# LIRA_DOCKER_IMAGE="quay.io/humancellatlas/secondary-analysis-lira:${LIRA_DOCKER_TAG}"
+
+PIPELINE_TOOLS_PREFIX="https://raw.githubusercontent.com/HumanCellAtlas/pipeline-tools/${PIPELINE_TOOLS_VERSION}"
+SERVICE=${SERVICE:-"lira"}
+MAX_CROMWELL_RETRIES=${MAX_CROMWELL_RETRIES:-"1"}
+
+# Cromwell URL - usually will be caas, but can be set to local environment
+CROMWELL_URL=${CROMWELL_URL:-"https://cromwell.${CAAS_ENVIRONMENT}.broadinstitute.org/api/workflows/v1"}
+
+COLLECTION_NAME=${COLLECTION_NAME:-"lira-${LIRA_ENVIRONMENT}-workflows"}
```

Comment:
`COLLECTION_NAME` is defined on line 136 so we can remove this

Reply:
Done.